README.md

```diff
@@ -84,9 +84,14 @@ The LIDAR was used to gather all the data used to map the environment. The raw d
 
 ### Cartographer
 <p>
-Cartographer is an algorithm/system used to get a representation of the local environment given useful sensor data. For localization and mapping of the local environment, SLAM is the main part. 
+The Cartographer system is derived from Google’s repository in GitHub where the system is an open source project, letting us to use it. This system’s main focus is to be a solution to the simultaneous localization and mapping (SLAM) problem that is about how one can do both localization and mapping at the same time. 
```

Comment:
This part needs rephrasing. "Letting us to use it." is not proper grammar.

Reply:
Ok